src/MyHealth/DayIndicator.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { FunctionComponent } from "react"
+import { View, Text, ViewStyle, TextStyle, StyleSheet } from "react-native"
+import dayjs from "dayjs"
+
+import { DateTimeUtils } from "../utils"
+import { Outlines, Colors, Typography, Spacing } from "../styles"
+import { CheckInStatus, DayLogData } from "./symptoms"
+
+interface DayIndicatorProps {
```

Comment:
```suggestion
interface CalendarDayProps {
```
I think we initially used the `Indicator` terminology because there was a naming conflict, so what do you think about `CalendarDay`?